clients/admin-ui/src/features/privacy-requests/PrivacyRequestsContainer.tsx

```diff
@@ -14,6 +14,7 @@ const ActionButtons = dynamic(
 
 const PrivacyRequestsContainer: React.FC = () => {
   const { processing } = useDSRErrorAlert();
+  const [revealPII, setRevealPII] = useState(false);
```

Comment:
@allisonking It wasn't too bad to convert it over to a `useState` so I went ahead and did it. There is a little prop drilling required for the table page due to the PII components being split between multiple pages. The details page was simple because both PII components are located in the same component.